src/dbnode/storage/bootstrap.go

```diff
@@ -243,7 +221,7 @@ func (m *bootstrapManager) bootstrap() error {
 		// an error returned.
 		for _, accumulator := range accmulators {
 			if err := accumulator.Close(); err != nil {
-				instrument.EmitAndLogInvariantViolation(m.opts.InstrumentOptions(),
+				instrument.EmitAndLogInvariantViolation(m.instrumentation.opts.InstrumentOptions(),
```

Comment:
Nit: IMHO it would be somewhat more natural for `bootstrapManager` to have it's own `opts`.